modules/sdk-coin-atom/src/atom.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  BaseCoin,
+  BitGoBase,
+  ExplanationResult,
+  KeyPair,
+  MPCAlgorithm,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  UnsignedTransaction,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '@bitgo/sdk-core';
+
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+export class Atom extends BaseCoin {
+  protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo);
+
+    if (!staticsCoin) {
+      throw new Error('missing required constructor parameter staticsCoin');
+    }
+
+    this._staticsCoin = staticsCoin;
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Atom(bitgo, staticsCoin);
+  }
+
+  /**
+   * Factor between the coin's base unit and its smallest subdivison
+   */
+  public getBaseFactor(): number {
+    return 1e6;
+  }
+
+  public getChain(): string {
+    return 'atom';
+  }
+
+  public getFamily(): string {
+    return 'atom';
+  }
+
+  public getFullName(): string {
+    return 'Cosmos';
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return true;
+  }
+
+  getMPCAlgorithm(): MPCAlgorithm {
+    return 'eddsa';
+  }
```

Comment:
should this be ecdsa?
also, can we have this be derived from static coin?

Reply:
Yes it should, and as for deriving from the static coin type not as simply as I can tell - we could make it an attribute of the StaticCoin type in the future if we'd like to do a bit of refactoring.